libraries/AP_Camera/LogStructure.h

```diff
@@ -38,8 +39,32 @@ struct PACKED log_Camera {
     uint16_t yaw;
 };
 
+// @LoggerMessage: MNT
+// @Description: Mount current and Target RPY information
+// @Field: TimeUS: Time since system startup
+// @Field: MR: current mount roll
+// @Field: MP: current mount pitch
+// @Field: MP: current mount yaw
+// @Field: TR: target mount roll
```

Comment:
I think we should make this message look more like the "ATT" message so perhaps change the columns to "DesRoll, Roll, DesPitch, Pitch, DesYaw, Yaw".  Let's make sure those are logged in degrees and we also need an instance field.